Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -69,8 +70,14 @@ xxx yyy
         public void ProcRunner_FailsOnTimeout()
         {
             // Arrange
+
+            // Previously the script used "TIMEOUT 2" to pause execution in
+            // the spawned process. However, the test was failing on some machines
+            // with the error "Input redirection is not supported, exiting the process immediately."
+            // Pinging a non-existent address with a timeout is an alternative 
+            // that works if the machine is connected to the internet
             string exeName = TestUtils.WriteBatchFileForTest(TestContext,
-@"TIMEOUT 2
+@"@ping 1.1.1.1 -n 1 -w 2000 >nul
```

Comment:
Just out of interest - this seems like a very strange way to replace "TIMEOUT 2".